keychain/src/types.rs

```diff
@@ -227,12 +228,13 @@ impl fmt::Display for Identifier {
 	}
 }
 
-#[derive(Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[derive(Default, Clone, PartialEq, Serialize, Deserialize, Zeroize)]
 pub struct BlindingFactor([u8; SECRET_KEY_SIZE]);
 
+// Intentionally empty `Default` implementation to prevent leakage.
```

Comment:
This line of comment confuse me a bit, is it talking about `Default` derive?  or `Debug`? because next line is Debug impl.

Reply:
Good catch! Obviously a typo. I meant to make blank Debug impl, not Default.

Reply:
Minor point but this got me thinking - can we do something other thank blank here?
We could print out `********` or `<REDACTED> `something, that way we can actually check the logs and know if we are doing we probably shouldn't. If its just blank we'll just never see it.

Also less likely that someone will come in, ignore the comment and "fix" the impl.

Reply:
Seems reasonable!